db/schema.rb

```diff
@@ -438,6 +438,7 @@
     t.string "session_id", limit: 255
     t.datetime "submitted_at", precision: nil
     t.datetime "last_graded_time", precision: nil, default: "2021-10-24 14:11:56"
+    t.datetime "timer_started_at"
```

Comment:
Just for checking, is this `precision: 6` in the actual DB schema (e.g. `timestamp(6)`)?  
If I recall correctly, if not specified, Rails now imposes `precision: 6` implicitly https://stackoverflow.com/a/71482301

For postgresql by default, there is no explicit bound on precision. The allowed range of p is from 0 to 6.
https://www.postgresql.org/docs/current/datatype-datetime.html#DATATYPE-DATETIME